tutorials/shaders/shader_reference/canvas_item_shader.rst

```diff
@@ -260,14 +263,14 @@ Below is an example of a light shader that takes a CanvasItem's normal map into
 +----------------------------------+------------------------------------------------------------------------------+
 | in float **LIGHT_ENERGY**        | Energy multiplier of Light.                                                  |
```

Comment:
All the references to "Light" I left alone. I am not familiar with 2D shaders - do these refer to the `LIGHT` built-in, or a 2D light node?

Reply:
Refers to a 2D light node, (technically a "light instance" I think)

Reply:
Changed these into links to Light2D and it's properties.